grease-cli/tests/llvm/func-ptr.llvm.cbl

```diff
@@ -1,12 +1,24 @@
 ; Copyright (c) Galois, Inc. 2024
 
+; Test the behavior of calls through symbolic function pointers. By default,
+; such calls are simply skipped, but --error-symbolic-fun-calls turns them
+; into a hard error.
+
+;; flags {"--symbol", "test"}
+;; go(prog)
+;; check "Skipped call to a symbolic function handle"
+;; ok()
+
 ;; flags {"--symbol", "test"}
+;; flags {"--error-symbolic-fun-calls"}
 ;; go(prog)
+;; check "Goal failed:"
+;; check "Failed to load function handle"
+;; check "Cannot resolve a symbolic pointer to a function handle"
+;; no_heuristic()
```

Comment:
If I understand correctly, this is running the same test case twice, but with different arguments (and a different set of checks). I assume this works because the two sets of semicolon-delimited lines have a newline in between them? If so, is this convention documented somewhere?

Reply:
> If I understand correctly, this is running the same test case twice, but with different arguments (and a different set of checks).

:+1:

> I assume this works because the two sets of semicolon-delimited lines have a newline in between them?

There's nothing special in the formatting - it could all be one big block. All of the `;;` comments are a single Lua program. This Lua program runs GREASE twice via `go`, resetting the built-in state of Oughta (e.g., `text`, output line / column tracking, matches) each time.

Let me know if that makes sense, and/or if there's a way to make it more clear!

Reply:
Ah. Perhaps it would be worth clarifying that `go` can be run multiple times within a single file in its documentation, then?